debian/control

```diff
@@ -3,15 +3,46 @@ Section: web
 Priority: optional
 Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
 Uploaders: Francois Marier <francois@debian.org>
-Build-Depends: debhelper (>= 9), python-all, dh-python, python-nose, python-nosexcover, python-tox, python-docutils, python-sphinx, python-sphinx-rtd-theme, python-repoze.sphinx.autointerface
+Build-Depends: debhelper (>= 9),
+               dh-python,
+               python-all,
+               python-configargparse,
+               python-dialog (>=3.2.2rc1),
```

Comment:
For the initial Debian package support, I'm focusing on Debian unstable/sid. Once that works, we can look at how much of an effort it will be to backport all of the dependencies.

I add the version number so that the Debian dependencies match the setup.py.